web/packages/teleport/src/services/userEvent/types.ts

```diff
@@ -101,6 +101,7 @@ export enum DiscoverEvent {
 export enum DiscoverEventResource {
   Server = 'DISCOVER_RESOURCE_SERVER',
   Kubernetes = 'DISCOVER_RESOURCE_KUBERNETES',
+  KubernetesEks = 'DISCOVER_RESOURCE_KUBERNETES_EKS',
```

Comment:
is this defined in our proto files? (look up the other defined events, and you'll see it's the values defined in a few protos)

we should put a comment to make it clear that these are values expected in the backend

also i'm not sure i'm seeing it used